cmd/lakectl/cmd/branch_revert.go

```diff
@@ -58,6 +61,7 @@ func init() {
 	AssignAutoConfirmFlag(branchRevertCmd.Flags())
 
 	branchRevertCmd.Flags().IntP(ParentNumberFlagName, "m", 0, "the parent number (starting from 1) of the mainline. The revert will reverse the change relative to the specified parent.")
+	branchRevertCmd.Flags().Bool(allowEmptyRevert, false, "allow revert with no changes")
```

Comment:
same here as part of the description where we instruct that we allow empty commit as part of the revert operation.